src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractServerTask.groovy

```diff
@@ -301,9 +301,10 @@ abstract class AbstractServerTask extends AbstractTask {
             serverEnvPath = server.serverEnvFile.getCanonicalPath()
         }
 
-        if (container != null && container.booleanValue()) {
-            // Set PROJECT_ROOT_NAME so it will be written in config file.
-            server.var."io.openliberty.tools.projectRoot" = project.getProjectDir().getAbsolutePath()
+        if (container != null && container.booleanValue() ||
+            System.getProperty(PROJECT_ROOT_NAME) != null) { // property set by CreateTask
```

Comment:
Is the comment `// property set by CreateTask` inaccurate? Since the property is not actually set by CreateTask, but it's set by DevTask when calling CreateTask.